src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs

```diff
@@ -40,6 +40,56 @@ public async Task InsertMutation(string dbQuery)
             SqlTestHelper.PerformTestEqualJsonStrings(expected, actual.ToString());
         }
 
+        /// <summary>
+        /// <code>Do: </code> Inserts new row and return all its columns
+        /// <code>Check: </code> A row is inserted in the table that has rows with default values as built_in methods.
+        /// it should insert it correctly with default values correctly handled by database.
+        /// current_date, current_timestamp, random_number, next_day have default value as built_in methods GETDATE(), NOW(), RAND(), DATEADD(), resp.
+        /// default_string_with_parenthesis has default value "()", default_function_string_with_parenthesis has default value "NOW()".
+        /// default_integer has default value 100, default_date_string has default value "1999-01-08 10:23:54".
+        /// Returned response would look like:
+        /// "createDefaultBuiltInFunction": {
+        ///   "current_date": "2023-12-15T16:24:48.267Z",
+        ///   "current_timestamp": "2023-12-15T16:24:48.267Z",
+        ///   "random_number": 0,
+        ///   "next_day": "2023-12-16T00:00:00.000Z",
+        ///   "default_string_with_paranthesis": "()",
+        ///   "default_function_string_with_paranthesis": "NOW()",
+        ///   "default_integer": 100,
+        ///   "default_date_string": "1999-01-08T10:23:54.000Z"
+        /// }
+        /// </summary>
+        public virtual async Task InsertMutationWithDefaultBuiltInFunctions()
+        {
+            string graphQLMutationName = "createDefaultBuiltInFunction";
+            string graphQLMutation = @"
+                mutation {
+                    createDefaultBuiltInFunction(item: { user_value: 1234 }) {
+                        current_date
+                        current_timestamp
+                        random_number
+                        next_day
+                        default_string_with_parenthesis
+                        default_function_string_with_parenthesis
+                        default_integer
+                        default_date_string
+                    }
+                }
+            ";
+
+            JsonElement result = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+
+            // Assert the values
```

Comment:
if you look at other tests, we query the database to find the json object and compare those, why not do the same here?

Reply:
mutation query already does that and returns the response. other tests we are checking if the value inserted are same as the response returned. we already have other tests conforming it. Here we wanted to tests if the default values are inserted or not. and this test is sufficient.
Furthermore, the value of dates returned by sql query and graphql are not same, so extra step would be required there as well to convert it into same format.

Reply:
i'll update the test to have both checks